README.md

```diff
@@ -18,7 +18,9 @@ $ yarn
 
 ## Run Application Server
 
+make sure to build the webpacks bundle.js before serving
 ```
+$ npm run build
```

Comment:
This maybe belongs in its own commit?

Reply:
Yeah I think also we can clean up the commit history here with an interactive rebase, and split them up logically

Reply:
I think here we need to be explicit about what needs to happen before we release changes. This adds a step to our release workflow where we need to run `npm build` and then commit the compiled assets in a release branch. Let's chat in the slack about this.